package.json

```diff
@@ -38,38 +40,34 @@
     "argv-parse": "^1.0.1",
     "babel-eslint": "^10.0.0",
     "babel-loader": "^8.0.0",
-    "compression-webpack-plugin": "~1.1.11",
     "cross-env": "^5.2.0",
-    "css-loader": "^0.23.1",
-    "cssnano": "^4.1.10",
+    "css-loader": "6.8.1",
+    "css-minimizer-webpack-plugin": "^4.2.2",
+    "cssnano": "^5.0.1",
     "dotenv": "^5.0.0",
     "eslint": "^6.7.2",
     "eslint-plugin-spellcheck": "0.0.17",
-    "expose-loader": "~0.6.0",
-    "extract-text-webpack-plugin": "^3.0.0",
-    "file-loader": "^0.9.0",
     "graphql": "^15.5.0",
     "highlight.js": "~9.14.0",
-    "node-sass": "^4.5.0",
-    "optimize-css-assets-webpack-plugin": "^3.2.0",
+    "node-sass": "8.0.0",
+    "path-browserify": "^1.0.1",
     "prettier": "^1.19.1",
     "pretty-format": "26.6.2",
     "raw-loader": "^0.5.1",
     "react-dnd-test-backend": "^9.4.0",
     "react-dnd-test-utils": "^9.4.0",
     "react-remarkable": "^1.1.3",
     "redux-mock-store": "^1.2.2",
-    "sass-loader": "~6.0.6",
-    "style-loader": "^0.13.1",
+    "sass": "~1.60.0",
+    "sass-loader": "10.2.0",
+    "style-loader": "^1.3.0",
     "stylelint": "^9.3.0",
     "stylelint-config-standard": "^18.0.0",
-    "uglifyjs-webpack-plugin": "^1.2.2",
-    "url-loader": "^1.0.1",
-    "webpack": "^3.4.1",
-    "webpack-bundle-analyzer": ">=3.3.2",
-    "webpack-dev-server-without-h2": "^2.11.8",
-    "webpack-stats-plugin": "^0.1.5",
     "tabbable": "~5.2.0",
-    "sass": "~1.60.0"
+    "url-loader": "4.1.1",
+    "webpack": "5.75.0",
+    "webpack-bundle-analyzer": "^4.5.0",
+    "webpack-cli": "5.0.1",
```

Comment:
what is this used for? I didn't see any `webpack-cli` calls in your PR, or does this also provide the `webpack` binary these days?
(also, too strict version req ;) )

Reply:
Since were using `npx webpack`, we need to have `webpack-cli`.
from https://webpack.js.org/api/cli :

> Warning
If you want to run webpack using npx please make sure you have webpack-cli installed.

Reply:
TIL, thanks!